src/PayPal/Api/Plan/BillingCycle/PricingScheme/FixedPrice.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Plan\BillingCycle\PricingScheme;
+
+use Swag\PayPal\PayPal\Api\Common\PayPalStruct;
+
+class FixedPrice extends PayPalStruct
```

Comment:
we already have a common struct for this case https://github.com/shopwareLabs/SwagPayPal/blob/master/src/PayPal/Api/Common/Value.php
please extend from this file und also change the ones extending from FixedPrice

Reply:
@mitelg it's not the same :D currency vs currencyCode :D

Reply:
changed to money